client/blocks/domain-to-plan-nudge/index.jsx

```diff
@@ -67,149 +58,42 @@ class DomainToPlanNudge extends Component {
-							{ translate( 'Upgrade to a Personal Plan and Save!' ) }
-						</h3>
-						<ul className="domain-to-plan-nudge__header-features">
-							<li>
-								<div className="domain-to-plan-nudge__header-features-item">
-									<Gridicon
-										className="domain-to-plan-nudge__header-features-item-checkmark"
-										icon="checkmark"
-										size={ 24 }
-									/>
-									{ translate( 'Remove all WordPress.com advertising from your website' ) }
-								</div>
-							</li>
-							<li>
-								<div className="domain-to-plan-nudge__header-features-item">
-									<Gridicon
-										className="domain-to-plan-nudge__header-features-item-checkmark"
-										icon="checkmark"
-										size={ 24 }
-									/>
-									{ translate( 'Get high quality live chat and priority email support' ) }
-								</div>
-							</li>
-							<li>
-								<div className="domain-to-plan-nudge__header-features-item">
-									<Gridicon
-										className="domain-to-plan-nudge__header-features-item-checkmark"
-										icon="checkmark"
-										size={ 24 }
-									/>
-									{ translate( 'Bundled with your domain for the best value!' ) }
-								</div>
-							</li>
-						</ul>
-					</div>
-				</div>
-				<div className="domain-to-plan-nudge__actions-group">
-					<div className="domain-to-plan-nudge__plan-price-group">
-						<div
-							className="domain-to-plan-nudge__discount-value">
-							{
-								translate( 'SAVE %(discount)s', {
-									args: {
-										discount: formatCurrency( rawDiscount, userCurrency )
-									}
-								} )
-							}
-						</div>
-						<PlanPrice
-							rawPrice={ rawPrice }
-							currencyCode={ userCurrency }
-							original
-						/>
-						<PlanPrice
-							rawPrice={ discountedRawPrice || rawPrice }
-							currencyCode={ userCurrency }
-							discounted
-						/>
-
-						<div className="domain-to-plan-nudge__plan-price-timeframe">
-							{ translate( 'for a one year subscription' ) }
-						</div>
-					</div>
-					<div className="domain-to-plan-nudge__upgrade-group">
-						<Button
-							className="domain-to-plan-nudge__upgrade-button"
-							onClick={ this.personalCheckout }
-							primary
-						>
-							{
-								translate( 'Upgrade Now for %s', {
-									args: formatCurrency( discountedRawPrice || rawPrice, userCurrency ),
-									comment: '%s will be replaced by a formatted price, i.e $9.9'
-								} )
-							}
-						</Button>
-					</div>
-				</div>
-			</DismissibleCard>
+			<Banner
+				callToAction={
+					translate( 'Upgrade for %s', {
+						args: formatCurrency( discountedRawPrice || rawPrice, userCurrency ),
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				event="domain_to_personal_nudge" //actually cta_name
+				dismissPreferenceName="domain-to-plan-nudge"
+				feature={ FEATURE_NO_ADS }
+				href={ `/checkout/${ siteId }/personal` }
+				list={
+					[
+						translate( 'No advertising' ),
```

Comment:
Alternate string suggestion: 
* `translate( 'No Advertising' )`&mdash; translations: **25**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domain-to-plan-nudge)